t/admin/upstream.t

```diff
@@ -235,7 +235,7 @@ GET /t
 GET /t
 --- error_code: 400
 --- response_body
-{"error_msg":"invalid configuration: property \"nodes\" validation failed: expect object to have at least 1 properties"}
+{"error_msg":"invalid configuration: property \"nodes\" validation failed: object matches none of the requireds"}
```

Comment:
I don't think it should be modified here.

Reply:
If not changed, the case cannot be passed.

or we need change `jsonschema`'s message